module/VuFind/tests/unit-tests/src/VuFindTest/Controller/Plugin/ResultScrollerTest.php

```diff
@@ -412,18 +418,20 @@ class ResultScrollerMock extends \VuFind\Controller\Plugin\ResultScroller
      */
     protected $testResults;
 
-    public function __construct($mockManager, $testResults)
+    public function __construct($mockManager, $testResults, $memory)
     {
-        parent::__construct(new Container('test'), $mockManager);
+        parent::__construct(new Container('test'), $mockManager, true, $memory);
         $this->testResults = $testResults;
     }
 
     /**
      * Stubbed
      *
-     * @return \VuFind\Search\Base\Results
+     * @param int $searchId Search ID
+     *
+     * @return ?\VuFind\Search\Base\Results
      */
-    protected function restoreLastSearch()
+    protected function restoreSearch(int $searchId): ?\VuFind\Search\Base\Results
```

Comment:
There's a comment elsewhere in this file that still refers to restoreLastSearch and should be updated.

Reply:
Updated.